src/components/sequencing/SequenceEditor.svelte

```diff
@@ -192,12 +191,12 @@
         EditorView.lineWrapping,
         EditorView.theme({ '.cm-gutter': { 'min-height': `${clientHeightGridRightTop}px` } }),
         lintGutter(),
-        compartmentSeqLanguage.of(setupLanguageSupport(sequenceCompletion(null, null, []))),
-        compartmentSeqLinter.of(sequenceLinter()),
+        compartmentSeqLanguage.of(setupLanguageSupport(get(sequenceAdaptation).autoComplete(null, null, []))),
+        compartmentSeqLinter.of(inputLinter()),
         compartmentSeqTooltip.of(sequenceTooltip()),
         EditorView.updateListener.of(debounce(sequenceUpdateListener, 250)),
         EditorView.updateListener.of(selectedCommandUpdateListener),
-        indentService.of(sequenceAutoIndent()),
+        indentService.of(get(sequenceAdaptation).autoIndent()),
```

Comment:
do you need to explicitly call `get` on this store or can you just do the `$` notation?

Reply:
also is it okay if this gets set only on `onMount` and not when `sequenceAdaptation` gets updated?

Reply:
Good catch, it should be whenever the adaptation updates. I'll change it to include the `$` as well!